README.md

```diff
@@ -279,6 +279,7 @@ mdsf init
 | kdlfmt               | [https://github.com/hougesen/kdlfmt](https://github.com/hougesen/kdlfmt)                                                     |
 | ktfmt                | [https://github.com/facebook/ktfmt](https://github.com/facebook/ktfmt)                                                       |
 | ktlint               | [https://github.com/pinterest/ktlint](https://github.com/pinterest/ktlint)                                                   |
+| kulala-fmt           | [https://github.com/mistweaverco/kulala-fmt](https://github.com/mistweaverco/kulala-fmt)                                     |
```

Comment:
I think I already raised the point, but I'm unsure in which issue, so please pardon me.

But having the list of linter without information about what they support looks strange to me.

I mean there is no way to know the .http format can be handled by kulala-fmt or that your tool supports HTTP format.

I see nothing in the code, or the test to know http fenced block could use kulala

Am I missing something?

Reply:
Please also consider 

- https://github.com/hougesen/mdsf/issues/490